Tools/CISupport/ews-build/steps.py

```diff
@@ -2276,7 +2276,6 @@ def propertiesToPassToTriggers(self, patch=True, pull_request=False):
             'platform',
             'fullPlatform',
             'architecture',
-            'additionalArguments',
```

Comment:
This is simply reverting previous attempt to fix this issue (https://commits.webkit.org/263195@main).